templates/edit_mod.html

```diff
@@ -173,17 +173,25 @@ <h2>Additional Authors</h2>
     </div>
     {% endif %}
 
-    <div class="container" style="margin-top: 2.5mm;">
-        <p>
-            <span class="col-md-10 form-group"><a target="_blank" href="/markdown">Markdown</a> is supported here.</span>
-            {% if not mod.ckan %}
-            <label>
-                <input id="ckan" name="ckan" style="margin-top: 8px;" type="checkbox"> Add mod to <a target="_blank" href="https://github.com/KSP-CKAN/CKAN">CKAN</a>
-                <span data-original-title="This makes it so that users can automate the installation of your mod. Your mod won't be added to CKAN until you publish it." class="glyphicon glyphicon-question-sign" data-toggle="tooltip" data-placement="left" title=""></span>
-            </label>
-            {% endif %}
-        </p>
-        <textarea class="form-control input-block-level" name="description" id="description" rows=25>{{ mod.description }}</textarea>
+    <div class="container">
+        <div class="row">
+            <div class="col-md-5"><a target="_blank" href="/markdown">Markdown</a> is supported here.</div>
+            <div class="col-md-5 form-group text-right">
+                <input id="ckan" name="ckan" type="checkbox"
+                    {% if mod.ckan %}
+                       checked="checked"
+                       {% if mod.published and not user.admin %}
+                           disabled="disabled"
+                       {% endif %}
+                    {% endif %}
+                />
+                <label for="ckan">Add mod to <a target="_blank" href="https://github.com/KSP-CKAN/CKAN">CKAN</a></label>
```

Comment:
```suggestion
                <label for="ckan">Request addition to <a target="_blank" href="https://github.com/KSP-CKAN/CKAN">CKAN</a></label>
```

I agree that something should signal that this process is not fully automated, but I'm not sure that a tooltip about a pull request and a repository will make any sense to the typical user. Maybe if the checkbox uses a different verb than "Add"?